components/Input/Input.css

```diff
@@ -0,0 +1,98 @@
+@import url(../../style.css);
+
+.input,
+.text-area {
+  background-color: var(--ternary-color);
+  border: 1px solid var(--primary-dark);
+  outline: none;
+  color: var(--secondary-color);
+  padding: 7px;
+  position: relative;
+}
+
+.input:hover,
+.input:focus,
+.text-area:hover,
+.text-area:focus {
+  border: 1px solid var(--primary-color);
+  background-color: var(--ternary-dark);
+}
+
+.input-validation-msg {
+  content: "wrong password!";
+  color: red;
+  display: block;
+  position: absolute;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+  appearance: none;
+
+  width: 1.15em;
+  height: 1.15em;
```

Comment:
This will inherit the same width and height in every project you use your component library.

Reply:
Yes Prajwal as it is a radio button I wanted it to have fixed size